Smartling.API/CloudLog/SmartlingCloudLogger.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Threading;
+using System.Threading.Tasks;
+using Smartling.Api.Model;
+using Smartling.Api.Threading;
+
+namespace Smartling.Api.CloudLog
+{
+  public class SmartlingCloudLogger : IDisposable
```

Comment:
This class will be used in custom log4net appended or in sitecore log appended. Just wanted to keep it logger agnostic and don't add new dependency into sdk.
But your thoughts are welcome.